static/js/tools/map/fetcher/geojson.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Fetch Geo Json data.
+ */
+
+import axios from "axios";
+import _ from "lodash";
+import { Dispatch } from "react";
+
+import { GeoJsonData, GeoJsonFeature } from "../../../chart/types";
+import { IPCC_PLACE_50_TYPE_DCID } from "../../../shared/constants";
+import { ChartDataType, ChartStoreAction } from "../chart_store";
+
+export const MANUAL_GEOJSON_DISTANCES = {
+  [IPCC_PLACE_50_TYPE_DCID]: 0.5,
+};
+
+export function fetchGeoJson(
+  parentPlace: string,
+  childType: string,
+  dispatch: Dispatch<ChartStoreAction>
+): void {
+  const action: ChartStoreAction = {
+    type: ChartDataType.geoJson,
+    error: null,
+    context: {
+      placeInfo: {
+        enclosingPlace: {
+          dcid: parentPlace,
+          name: "",
+        },
+        enclosedPlaceType: childType,
+      },
+    },
+  };
+  axios
+    .get(
+      `/api/choropleth/geojson?placeDcid=${parentPlace}&placeType=${childType}`
+    )
+    .then((resp) => {
+      if (_.isEmpty(resp.data)) {
+        action.error = "error fetching geo json data";
+      } else {
+        action.payload = resp.data as GeoJsonData;
+      }
+      dispatch(action);
+      console.log("geojson dispatched");
+    })
+    .catch(() => {
+      action.error = "error fetching geo json data";
+      dispatch(action);
+    });
+}
+
+export function getGeoJsonDataFeatures(
```

Comment:
do we still want to keep this function in the fetcher file? It looks like all the other fetcher files are just purely a data fetch function

Reply:
Moved it to util.ts